### PR TITLE
Add several basic policies for the 2i2c hubs

### PR DESCRIPTION
**Pull request by choldgraf**
This adds three policies to our infrastructure documentation. We've discussed each of them over time but hadn't yet put them in a public place. I believe that we still need to improve them all, and in some cases may need a lawyer to give an official stamp of approval, so for now I have noted each of them as non-legally-binder policies that are there to set expectations.

Unclear if we can close out any of these issues so I'll cross-ref them below:

- ref: https://github.com/2i2c-org/infrastructure/issues/342
- ref: https://github.com/2i2c-org/docs/issues/139
- ref: https://github.com/2i2c-org/team-compass/issues/388

### CODE REVIEW

File: policy/privacy.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+2i2c does not retain any identifiable information about its users or the information they create or place in the infrastructure we run.
+We require _access_ to the information our infrastructure while it is running, but do not retain this information when we stop working with a community.
+
+## While we run infrastructure
+
+- 2i2c's engineering team requires access to the filesystems of any user on our infrastructure for debugging and maintenance purposes.
+  - We do not retain or share this information for any purpose.
+- 2i2c may have access to personally-identifiable information that is used for _authenticating_ users (e.g. e-mail log-ins).
+  - We do not retain or share this information for any purpose.
+- We do not change or delete any user data on a hub without the consent of the hub's Community Representative.
```
Comment (sgibson91) [inline policy/privacy.md:20]: Maybe link to the definition of Community Representative here

File: policy/acceptable-use.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+- You must not use 2i2c Managed Infrastructure and/or network resources in conjunction with the execution of programs, software, processes, or automated transaction-based commands that are intended to disrupt (or that could reasonably be expected to disrupt) other computer or network users, or damage or degrade performance, software or hardware components of a system.
+- Do not use the 2i2c Managed Infrastructure to distribute or facilitate the sending of unsolicited or unlawful (i) email or other messages, or (ii) promotions of any kind;
+- Do not use the 2i2c Managed Infrastructure to engage in or promote any other fraudulent, deceptive or illegal activities.
+- Do not use the 2i2c Managed Infrastructure to process, store or transmit material, including any Customer Data, in violation of any Law or any third party rights, including without limitation privacy rights;
+- Do not use the 2i2c Managed Infrastructure in any circumstances where failure could lead to death, personal injury or environmental damage, and you further acknowledge that the 2i2c Managed Infrastructure is not designed or intended for such use.
```
Comment (sgibson91) [inline policy/acceptable-use.md:32]: I would like us to explicitly mention crypto-mining as an activity that contravenes our acceptable use policy. I think we allude to it with "automated transaction-based commands" - but I would like to be crystal clear that we will not look kindly upon users found crypto-mining

**Comment by choldgraf** [comment]
Both comments addressed - thanks @sgibson91

**Comment by sgibson91** [comment]
I haven't given an explicit approving review btw, because I think more eyes from the team would be good here. But if one doesn't come in within an appropriate timeframe, 👍🏻 from me to merge.

**Comment by choldgraf** [comment]
I've also just now pinged CS&S to take a look at these docs, since they might be needed for any "official / legal" policies that we have. So let's give it a few days and re-visit next week if nobody has provided another review since then. @sgibson91 if you "approve" I will still hold off until then before merging or moving forward.

**Review by sgibson91 — APPROVED** [review]

**Comment by choldgraf** [comment]
FYI I heard from Paige at CS&S that they're OK with us posting this now if we wish. I've also asked if they can take a closer look but they said we can disentangle making these public vs. making them "legally official"

**Comment by choldgraf** [comment]
OK gonna merge this one in so that we have something to point people to! We can iterate from there

**Pull request merged**